/*****************************************************************
* terraform-provider-vcloud-director
* Copyright (c) 2017 VMware, Inc. All Rights Reserved.
* SPDX-License-Identifier: BSD-2-Clause
******************************************************************/


syntax = "proto3";



package proto;

message CatalogUploadMediaInfo{
        string catalog_name =1 ;
        string file_path = 2 ;
        string item_name = 3;
        
}


message CatalogUploadMediaResult{
        bool created = 1;
}



message CatalogUploadOvaInfo{
        string catalog_name =1 ;
        string file_path = 2 ;
        string item_name = 3;
        
}

message CatalogCheckResolvedInfo{
        string catalog_name =1 ;
        string item_name = 2;
        
}

message CatalogUploadOvaResult{
        bool created = 1;
}


message DeleteCatalogItemInfo{
        string catalog_name =1 ;
        string file_path = 2 ;
        string item_name = 3;
        
}


message DeleteCatalogItemResult{
        bool deleted = 1;
}


message IsPresentCatalogItemInfo{
        string catalog_name =1 ;
        string item_name = 2;
        
}


message IsPresentCatalogItemResult{
        bool present = 1;
}


message CaptureVAppInfo{
        string catalog_name =1 ;
        string item_name = 2;
        string vapp_name = 3;
        string vdc_name = 4;
        string description = 5;
        bool customize_on_instantiate =6;
        
}


message CaptureVAppResult{
        bool captured = 1;
}

